modules/costs/lib/costs/engine.rb

```diff
@@ -147,6 +147,7 @@ class Engine < ::Rails::Engine
            { controller: "/my/time_tracking", action: "index" },
            after: :my_page,
            caption: :label_my_time_tracking,
+           if: ->(*) { User.current.allowed_in_any_project?(:log_own_time) },
```

Comment:
As written in the ticket as well, this would not be sufficient. The `log_time` permission would enable time tracking also for yourself anywhere in the appliction. And it is not set up as a requirement, that the permissions are given together.

```suggestion
           if: ->(*) { User.current.allowed_in_any_project?(:log_own_time) || User.current.allowed_in_any_project?(:log_time) },
```